modelforge/potential/models.py

```diff
@@ -971,6 +996,11 @@ def __init__(
         super().__init__()
         from modelforge.utils.units import _convert_str_to_unit
 
+        if model_seed:
+            import torch
+
+            torch.manual_seed(model_seed)
```

Comment:
I'll note that, since this doesn't seem to really do anything at the moment, we could probably hold off on adding this as a user modifiable parameter (i.e., sticking it in the potential pydantic model)

Reply:
This change was motivated by a failure in this [test](https://github.com/choderalab/modelforge/blob/a4b6cda0a9a340b95c7a2df868147a22d4eaa73e/modelforge/tests/test_models.py#L481) at this assert [statment](https://github.com/choderalab/modelforge/blob/a4b6cda0a9a340b95c7a2df868147a22d4eaa73e/modelforge/tests/test_models.py#L531).

In this test, we were setting the seed manually for each `model_inference = NeuralNetworkPotentialFactory.generate_potential()` method call. But this resulted in failures for two potentials (can't remember which). I agree that this might not be the best motivation for this change, and it is only useful for tests, but that's also the reason why I wouldn't expose it in the TOML file and keep the default as `None`.

Reply:
Sounds reasonable.  This should be ultimately used by PyTorch lightning, so good to have the option to set.